delayed-listener/deploy.sh

```diff
@@ -2,12 +2,18 @@
 
 set -e -x
 
+if [[ $1 = "--solution" ]]; then
+  MVN_PROFILE="-P solution"
+  APP_LABEL=",solution=true"
+fi
+
+
 APP=delayed-listener
 
 oc project myproject
 
 oc new-build --binary --name=${APP} -l app=${APP}
-mvn clean package -DskipTests=true
+mvn clean package -DskipTests=true ${MVN_PROFILE}
 oc start-build ${APP} --from-dir=. --follow
-oc new-app ${APP} -l app=${APP},hystrix.enabled=true,project=workshop
+oc new-app ${APP} -l app=${APP},hystrix.enabled=true,project=workshop${APP_LABEL}
```

Comment:
Out of curiosity, why labeling the project with `solution=true`?

Reply:
So that we you deploy a solution, to know that the solution has been deployed on OpenShift... I saw that on Clement's workshop stuff but there it works better because it's more automated with fabric8 maven plugin.